### PR TITLE
feat(sdk): add object store.

### PR DESCRIPTION
**Pull request by Yangqing**

### CODE REVIEW

File: leptonai/cli/objectstore.py [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    console,
+    click_group,
+    get_connection_or_die,
+    explain_response,
+    sizeof_fmt,
+)
+from leptonai.api import objectstore as api
+
+
+_max_upload_file_size_limit = 4995 * 1024 * 1024
```
Comment (bddppq) [inline leptonai/cli/objectstore.py:20]: out of curiosity, why 4995 such odd number

File: leptonai/cli/objectstore.py [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+        " takes no effect."
+    ),
+    is_flag=True,
+)
+def get(key, file, bucket, return_url):
+    """
+    Gets the object with the given key.
+    """
+    if not file:
+        file = os.path.basename(key)
```
Comment (bddppq) [inline leptonai/cli/objectstore.py:49]: shall we use key or just the basename of key?

**Review by bddppq — APPROVED** [review]
LGTM

maybe we can add progress bar later

File: leptonai/cli/objectstore.py (hunk 1 above)
Comment (Yangqing) [reply leptonai/cli/objectstore.py:20]: It's 5MB smaller than 5GB as set by R2 https://developers.cloudflare.com/r2/reference/limits/ haha

File: leptonai/cli/objectstore.py (hunk 2 above)
Comment (Yangqing) [reply leptonai/cli/objectstore.py:49]: For convenience purposes I find myself always wanting to download and then put in the current folder. I guess user is recommended to actually explicitly specify file path.

**Pull request merged**